packages/astro/src/core/errors/errors-data.ts

```diff
@@ -1365,6 +1365,40 @@ export const FontFamilyNotFound = {
 	hint: 'This is often caused by a typo. Check that your Font component is using a `cssVariable` specified in your config.',
 } satisfies ErrorData;
 
+/**
+ * @docs
+ * @message No parser found for file `example.md`. Pass a parser to the `file` loader (e.g. `parser: mdx`).
```

Comment:
Can we instead use an example that might actually happen, because I believe you shouldn't *have* to ever pass an MDX parser to the built-in `file` loader? (If our MDX parser wasn't working, I think that would be a bug?) Similarly, should the file loader EVER be unable to parse a Markdown file automatically? (`example.md`)? Wouldn't that also be a bug?

The examples we give in docs are a CSV parser and a TOML parser, so these are more realistic examples:

https://docs.astro.build/en/guides/content-collections/#parser-function

Reply:
Good catch, you're right, so those examples aren't realistic. I’ll switch to something that really needs a custom parser, like CSV or TOML

Reply:
updated at message
```
No parser found for file `users.csv`. Pass a parser to the `file` loader (e.g. `parser: csv`).'
```